spec/c04matrices/generate/display_lowerTriangle_ofMatrixSpec.js

```diff
@@ -0,0 +1,15 @@
+const lowerTriangleOfMatrix = require('../../../src/c04matrices/generate/display_lowerTriangle_ofMatrix')
+
+describe('Lower Triangle of Matrix', function () {
+
+    it('Should print a lower triangle of matrix ', function () {
+        let matrix = [[1, 2, 3], [4, 5, 6], [7, 8, 9]]
```

Comment:
Format matrix on multiple line.
```javascript
let matrix = [[1, 2, 3], 
              [4, 5, 6], 
              [7, 8, 9]]
```

Also do this change in the other places in all files from this PR.